README.md

```diff
@@ -119,24 +119,24 @@ A typical case is aggregating results from jdbc input plugin.
 * You can do that using `push_previous_map_as_event` aggregate plugin option :
 ``` ruby
      filter {
-		 aggregate {
-		     task_id => "%{country_name}"
-		     code => "
-		     	map['tags'] ||= ['aggregated']
-		     	map['town_name'] ||= []
-		     	event.to_hash.each do |key,value|
-		     		map[key] = value unless map.has_key?(key)
-		     		map[key] << value if map[key].is_a?(Array)
-		     	end
-		     "
-		     push_previous_map_as_event => true
-		     timeout => 5
-		 }
-
-		 if "aggregated" not in [tags] {
-		 	drop {}
-		 }
-	 }
+         aggregate {
+             task_id => "%{country_name}"
+             code => "
+                map['tags'] ||= ['aggregated']
+                map['town_name'] ||= []
+                event.to_hash.each do |key,value|
+                    map[key] = value unless map.has_key?(key)
+                    map[key] << value if map[key].is_a?(Array)
+                end
+             "
+             push_previous_map_as_event => true
+             timeout => 5
+         }
+
+         if "aggregated" not in [tags] {
+            drop {}
+         }
+     }
 ```
```

Comment:
I added an example where I count user clicks in events. The example has all 3 new options. I updated example 3 to be example 4